### DISCUSSION TITLE
ImageUpscaleBy - can't find it!?

### DISCUSSION THREAD

**Discussion by dfl** [comfyanonymous/ComfyUI#2270] [General]
I'm following this video workflow, and it references ImageUpscaleBy but I can't find it anywhere in my installation or any custom node...  can anyone help me? Did it disappear or change names?

https://youtu.be/7m9ZZFU3HWo?si=QVIuNVSw9ZovDi8S&t=1318

see also:
https://www.reddit.com/media?url=https%3A%2F%2Fpreview.redd.it%2Frwf15nyp0vfb1.png%3Fwidth%3D1330%26format%3Dpng%26auto%3Dwebp%26s%3D88073ba5525f5bafac3dc51864ecd7f6df77d259

**Comment by ltdrdata** [comment]
It means your ComfyUI is outdated very much.

**Reply by dfl** [reply]
I just installed it this morning 🤷‍♂️

**Reply by ltdrdata** [reply]
Is it correct to install it via git clone, not Pinokio?

**Reply by dfl** [reply]
yes git clone and fetch/pull.

**Reply by ltdrdata** [reply]
One more thing. The name isn't `ImageUpscaleBy` it's `ImageScaleBy`

**Reply by dfl** [reply]
doh!🤦‍♂️ thanks
